distributions/aws/istio-ingress/overlays/cognito/params.env

```diff
@@ -2,4 +2,3 @@ CognitoUserPoolArn=
 CognitoAppClientId=
 CognitoUserPoolDomain=
 certArn=
-loadBalancerScheme=internet-facing
```

Comment:
can you remove from cognito readme steps before merging? step 3 of https://github.com/awslabs/kubeflow-manifests/tree/v1.3-branch/distributions/aws/examples/cognito#40-configure-ingress